### PR TITLE
Publish the sharedFx+sharedHost installer to azure blob storage.

### PR DESCRIPTION
**Pull request by Sridhar-MS**
Remove some dead code.

cc: @piotrpMSFT

---
This change is [Reviewable](https://reviewable.io/reviews/dotnet/cli/1996)

### CODE REVIEW

**Comment by Sridhar-MS** [comment]
@dotnet-bot test Windows_2016 x64 Release Build please

File: scripts/dotnet-cli-build/PublishTargets.cs [hunk 1]
```diff
@@ -83,6 +84,20 @@ public static BuildTargetResult PublishSdkInstallerFile(BuildTargetContext c)
         }
 
         [Target]
+        [BuildPlatforms(BuildPlatform.Windows, BuildPlatform.OSX)]//, BuildPlatform.Ubuntu)]
+        public static BuildTargetResult PublishSharedFxInstallerFile(BuildTargetContext c)
```
Comment (MichaelSimons) [inline scripts/dotnet-cli-build/PublishTargets.cs]: Consider refactoring the shared logic that this method has in common with PublishSdkInstallerFile into a common method.

File: scripts/dotnet-cli-build/PublishTargets.cs [hunk 2]
```diff
@@ -83,6 +84,20 @@ public static BuildTargetResult PublishSdkInstallerFile(BuildTargetContext c)
         }
 
         [Target]
+        [BuildPlatforms(BuildPlatform.Windows, BuildPlatform.OSX)]//, BuildPlatform.Ubuntu)]
```
Comment (brthor) [inline scripts/dotnet-cli-build/PublishTargets.cs]: no need for the comment here, this  isn't needed on ubuntu

File: scripts/dotnet-cli-build/PublishTargets.cs [hunk 3]
```diff
@@ -83,6 +84,20 @@ public static BuildTargetResult PublishSdkInstallerFile(BuildTargetContext c)
...
+        public static BuildTargetResult PublishSharedFxInstallerFile(BuildTargetContext c)
+        {
+            var installerFile = c.BuildContext.Get<string>("CombinedFrameworkHostInstallerFile");
```
Comment (brthor) [inline scripts/dotnet-cli-build/PublishTargets.cs]: Should the name of the function match what is being published here?

`PublishCombinedFrameworkHostInstallerFile`

**Pull request closed without merging**